src/reducers/revisions/status.js

```diff
@@ -0,0 +1,10 @@
+export default ( state = 'ready', action ) => {
+	switch ( action.type ) {
+		case 'REVISIONS_FETCHING':
+			return 'fetching';
+		case 'REVISIONS_SET':
```

Comment:
@dayllanmaza 
Was this supposed to be `REVISIONS_READY` ?

Reply:
Yes. Fixed